src/clj/rems/css/styles.clj

```diff
@@ -35,7 +35,7 @@
       (or attr-value (recur (rest attr-names))))))
 
 (def content-width (u/px 1200))
-(def logo-height-menu (u/px 40))
+(def logo-height-menu (u/px 50))
```

Comment:
So you both remove the padding and make it a big higher? Ok. Please add a note to the changelog so that users will know to double-check their theme when upgrading.

Reply:
That is essentially what was in the screenshot

Reply:
I can do that, sure

Reply:
Why is this increased? There is no need for it?

Reply:
Also if you increase the logo height should you not increase the menu height too or some content like LBR logo can be cropped?

Reply:
done

Reply:
@Macroz "There is no need for it?"?
I have demoed the screenshot in the rems chat

Reply:
Well here is my thinking.

I thought you said you were going to just quickly remove the unnecessary padding, but now I see you also increase the height of the logo forcing an increase in the size of the navbar. I don't see you taking into account the increased height of the navbar in the rest of the code though. Did you check that LBR theme works right after this because we have extra padding for the body so the navbar won't go over the page contents/hero logo?

I think we don't need to increase this logo height here at all, but should soncider it and any further changes with taking into account the needs to customize the logo height, customize the logo paddings, making a more responsive navbar, fixing navbar layout etc. I.e. what I suggested in for a good ticket on further tuning. For a quick improvement I'm saying we don't need this.

Reply:
<img width="1338" alt="Screenshot 2021-02-17 at 10 45 11" src="https://user-images.githubusercontent.com/7344718/108178543-3903dc80-710d-11eb-8319-480db06e5a17.png">

Reply:
Here is also in a picture what I mean about LBR and any theme with similar setup because the padding was not adjusted. Left is before, right is after height increase.
![navbar_height_problem](https://user-images.githubusercontent.com/823661/108181486-a9f8c380-7110-11eb-889f-4fe0d5a16f25.png)

Reply:
Ok, so I have double checked and the increase in menu does not seem to break anything

Reply:
Right, Ok, I will remove it